akka-slf4j/src/main/scala/akka/event/slf4j/Slf4jLogger.scala

```diff
@@ -104,7 +104,10 @@ class Slf4jLogger extends Actor with SLF4JLogging with RequiresMessageQueue[Logg
       }
 
     case InitializeLogger(_) =>
-      log.info("Slf4jLogger started")
+      if (log.isInfoEnabled) {
+        Thread.sleep(30L)
+        log.info("Slf4jLogger started")
+      }
```

Comment:
This doesn't look as a reliable work around. 

Do we know what the root cause is? Isn't it possible to log immediately after the first use of `LoggerFactory.getLogger`?

Reply:
Yes, another reliable one is to remove this line of code, which I think is better

Reply:
Root cause is probably, I guess, we're trying to invoke logger before its initialization finished.

Reply:
We could do that, but still, couldn't it be triggered by some other logging happening right after? Seems very fragile in Slf4j if it depends on timing for initialization.

Reply:
Yes, if need to trigger, maybe also need some approach to do the synchronize.